src/applications/personalization/appointments/actions/index.js

```diff
@@ -84,17 +84,23 @@ export function fetchConfirmedFutureAppointments() {
       .startOf('day')
       .toISOString();
 
+    // Maximum number of days you can schedule an appointment in advance in VAOS
+    const endDate = moment()
```

Comment:
I know we are moving away from using `moment`, but for now I just want to ensure parity with the vaos tool.